src/main/scala/com.snowplowanalytics.sauna/Sauna.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2015 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.sauna
+
+import java.io.File
+
+import awscala.dynamodbv2.DynamoDB
+import awscala.s3.S3
+import awscala.sqs.SQS
+import awscala.{Credentials, Region}
+
+import com.snowplowanalytics.sauna.loggers._
```

Comment:
Fair enough. But if you take a look at the other source files, we explicitly marking each lib: https://github.com/snowplow/snowplow/blob/master/3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EventEnrichments.scala#L30-L32